docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+# Copy this into a file called `docker-compose.yml`
+version: "3"
+
+services:
+    prod:
+        image: todo-app:prod
+        env_file:
+            - ./.env
+        build:
+            context: .
+            dockerfile: Dockerfile
+            target: production
+        ports:
+            - "5000:5000"
+        environment:
+            FLASK_ENV: production
+    dev:
+        image: todo-app:dev
+        env_file:
+            - ./.env
+        build:
+            context: .
+            dockerfile: Dockerfile
+            target: development
+        ports:
+            - "5001:5001"
+        volumes:
+            - .:/todo_app
```

Comment:
Possibly a change, but this volume isn't quite working as intended; your working directory in the container is `app` rather than `todo_app` - you might be meaning to use `./todo_app:/app/todo_app` if you want to match the bind mount you've specified in the Readme for running it without Docker Compose